api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1314,11 +1324,6 @@ public function publishPublication(Request $illuminateRequest): JsonResponse
             ], Response::HTTP_FORBIDDEN);
         }
 
-        if ($publication->getData('status') === PKPSubmission::STATUS_PUBLISHED) {
```

Comment:
This doesn't look right to me -- it should never be necessary to call `publishPublication` on already-published content.